app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,12 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
Since `Dwarf` does not implement all abstract methods of the `Player` class, `Dwarf` should also be an abstract class. Inherit it from `ABC` additionally to indicate that